ui/packages/catalog-ui-search/src/main/webapp/js/CQLUtils.js

```diff
@@ -334,8 +334,10 @@ function arrayFromPolygonWkt(wkt) {
   if (wkt.startsWith('POLYGON')) {
     const polygon = wkt.match(/\(\([^\(\)]+\)\)/g)
     return polygon.length === 1 ? arrayFromPartialWkt(polygon[0]) : []
+  } else if (wkt.startsWith('LINESTRING')) {
```

Comment:
The name of this function feels a little off if it handles Linestrings as well as Polygons.